The program did't save the changed version of the preferences structure.

Antonio Ospite <ospite@studenti.unina.it>

diff -bpruN uae-0.8.25/src/gtkui.c uae-0.8.25_patched/src/gtkui.c
--- uae-0.8.25/src/gtkui.c	2006-06-08 23:57:36.000000000 +0200
+++ uae-0.8.25_patched/src/gtkui.c	2006-06-09 09:31:08.000000000 +0200
@@ -282,7 +282,7 @@ static void save_config (void)
 	write_log ("Error saving options file!\n");
 	return;
     }
-    save_options (f, &currprefs);
+    save_options (f, &changed_prefs);
     fclose (f);
 }
 
